### PR TITLE
Fixes for the 2023 revision of quizzes

### PR DESCRIPTION
**Pull request by lkocman**
* Micro chameleon ->  Brookesia micra
* No DE is selected by default
* Header fix for CZ variant (one side had EN)

### CODE REVIEW

**Comment by lkocman** [comment]
Feedback is welcome!

File: flyers/opensuse_quiz/opensuse_quiz_2023_cz.pdf
Comment (baierjan) [inline flyers/opensuse_quiz/opensuse_quiz_2023_cz.pdf:1]: 1. Co je hlavím cílem projektu -> hlav**n**ím
9. trasakční aktualizace -> tra**n**sakční aktualizace

**Comment by baierjan** [comment]
Maybe it is time to include Slowroll into options for question 4?

For a hardcore variant, I always wondered why `ext4` is not an option for question 11 (or maybe `ext3` to make it more obvious it is not the correct option).

**Comment by ddemaio** [comment]
Those changes made look good to me @lkocman.

**Comment by lkocman** [comment]
Slowroll, is in a concept phase right now. I have not so good experience with making early concepts official.

File: flyers/opensuse_quiz/opensuse_quiz_2023_cz.pdf
Comment (lkocman) [reply flyers/opensuse_quiz/opensuse_quiz_2023_cz.pdf:1]: Thank you!

**Comment by lkocman** [comment]
@baierjan Not having ext4 in general is a reason to not confuse users :-) I think you just answered to your question.

The only considerable in our list would be ZFS, as there is a releated project, but I think that license incompability is known issue, and efforts are usually outside of the main project.

Simple and honest answer is I asked AI to make it obvious, and AI provided ;-)

**Comment by lkocman** [comment]
Merging. Thanks to all involved!

**Pull request merged**